runtime/Swift/Sources/Antlr4/misc/BitSet.swift

```diff
@@ -645,41 +645,7 @@ public class BitSet: Hashable, CustomStringConvertible {
     }
 
     public static func numberOfTrailingZeros(_ i: Int64) -> Int {
-        // HD, Figure 5-14
-        var x: Int32, y: Int32
-        if i == 0 {
-            return 64
-        }
-        var n: Int32 = 63
-        y = Int32(truncatingIfNeeded: i)
-        if y != 0 {
-            n = n - 32
-            x = y
-        } else {
-            x = Int32(truncatingIfNeeded: i >>> 32)
-        }
-
-        y = x << 16
-        if y != 0 {
-            n = n - 16
-            x = y
-        }
-        y = x << 8
-        if y != 0 {
-            n = n - 8
-            x = y
-        }
-        y = x << 4
-        if y != 0 {
-            n = n - 4
-            x = y
-        }
-        y = x << 2
-        if y != 0 {
-            n = n - 2
-            x = y
-        }
-        return Int(n - ((x << 1) >>> 31))
+        return i.trailingZeroBitCount
```

Comment:
Is containing function required at all? `numberOfTrailingZeros` can be replaced with `trailingZeroBitCount`

Reply:
I have not yet changed any public API (which these functions are) but as you are requesting it, I will inline these and remove them.

Reply:
Anyway we've changed a lot of public API in 4.10 and removed outdated function. Awaiting @parrt for the final decision.

Reply:
What duplications are we talking about? Sorry I'm trying to review lots of different things.

Reply:
Nevermind, @Dante-Broggi has already removed useless public API functions.